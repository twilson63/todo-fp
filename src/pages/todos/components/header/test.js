/* global test, expect */
import React from 'react'
import { shallow } from 'enzyme'
import Header from './'

test('Header Component renders correctly', done => {
  const props = {}
  const result = shallow(
    <Header {...props} />
  )

  expect(result).toMatchSnapshot()
  done()
})
