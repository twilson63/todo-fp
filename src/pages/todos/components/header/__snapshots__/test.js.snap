exports[`test Header Component renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <header
    className="header">
    <h1>
        todos
    </h1>
    <withState(withHandlers(NewTodo)) />
</header>,
  "nodes": Array [
    <header
      className="header">
      <h1>
            todos
      </h1>
      <withState(withHandlers(NewTodo)) />
</header>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Header />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <header
          className="header">
          <h1>
                    todos
          </h1>
          <withState(withHandlers(NewTodo)) />
</header>,
        "_debugID": 2,
        "_renderedOutput": <header
          className="header">
          <h1>
                    todos
          </h1>
          <withState(withHandlers(NewTodo)) />
</header>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Header />,
}
`;
